Bottom: f1d1a1b1d5ea452cf91c90379dcd0e4362b2a21b
Top:    328aa60e362e607ccf1653397f6b91fa58898607
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-03-02 16:13:43 -0300

Refresh of template-support.patch

---

diff --git a/include/gcc-cp-interface.h b/include/gcc-cp-interface.h
index af3b786..b5d3095 100644
--- a/include/gcc-cp-interface.h
+++ b/include/gcc-cp-interface.h
@@ -41,7 +41,7 @@ struct gcc_cp_context;
 
 enum gcc_cp_api_version
 {
-  GCC_CP_FE_VERSION_0 = 0xffffffff-4
+  GCC_CP_FE_VERSION_0 = 0xffffffff-5
 };
 
 /* Qualifiers.  */
